app/controllers/catalog_controller.rb

```diff
@@ -139,6 +139,7 @@ class CatalogController < ApplicationController
       label: 'Publication/Creation',
       helper_method: :multiple_values_new_line)
     config.add_index_field 'format_ssim', label: 'Type'
+    config.add_index_field 'edition_tsim', label: 'Edition', helper_method: :multiple_values_new_line
```

Comment:
So, just grepping for "tsim" vs. "tesim", "tsim" has 59 results and "tesim" has 532 results. I think the difference is that  "tesim" uses English language stemming, and I think since our user base is mostly English-speaking that would make sense for this field as well.

Maybe it would make sense to merge this as-is, and then have a ticket, possibly for post-phase-1, to make the use of tesim rather than tsim consistent throughout the application?

Reply:
@maxkadel Thanks for pointing that out. That would definitely require a ticket, since @eporter23 is maintaining a ["source of truth" document](https://docs.google.com/spreadsheets/d/1jSqgNd6HnVJdJMrrShAofkmK8uGgHGdlKODmZOL5NfY/edit#gid=1979033236) that is referenced by the stakeholders as to what is used in the application.